api/nakadi-event-bus-api.yaml

```diff
@@ -392,6 +394,46 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /event-types/{name}/effective_schema:
```

Comment:
The alternative would be to add a new field to the event-type. We would have to remember updating it every time something changes and also do some data migration in case we change the structure of the effective schema.

Regarding second point, it's just that it's my first contribution to the api. I'll change it.

@jkliff do you have any opinions on this one?